service/tablet_allocator.cc

```diff
@@ -763,14 +766,19 @@ class load_balancer {
         , _skiplist(std::move(skiplist))
     { }
 
+    rack_list_colocation ongoing_rack_list_colocation() const {
```

Comment:
bool_class is not useful for return types, because the name is in the function name.